src/modules/workspaces/repositories/team.repository.ts

```diff
@@ -135,11 +135,15 @@ export class SequelizeWorkspaceTeamRepository {
   async addUserToTeam(
     teamId: WorkspaceTeamAttributes['id'],
     userUuid: UserAttributes['uuid'],
+    options?: { transaction: Transaction },
   ): Promise<WorkspaceTeamUser | null> {
-    const teamUser = await this.teamUserModel.create({
-      teamId,
-      memberId: userUuid,
-    });
+    const teamUser = await this.teamUserModel.create(
+      {
+        teamId,
+        memberId: userUuid,
+      },
+      { ...options },
```

Comment:
By the way, does passing options directly not work?

`.create({...}, options)`

instead of

`.create({...}, {...options})`